requests/sessions.py

```diff
@@ -736,8 +736,10 @@ def session():
     """
     Returns a :class:`Session` for context-management.
 
+    This method is kept for backwards compatibility. For new code,
+    initialize :class:`Session` directly.
+
     :rtype: Session
     """
-    warnings.warn("'requests.session' is depreciated, use 'requests.Session' instead",
-                  DeprecationWarning, stacklevel=2)
+
```

Comment:
Please remove the extraneous new-line here.

Reply:
There are a few other methods in `sessions.py` with an extra newline (`post`, `put`, `patch`, `delete`, `send`). Should these be changed as well?